src/cloud/paths/experimental_sampledata_buckets_members.yml

```diff
@@ -0,0 +1,44 @@
+post:
+  operationId: GetDemoDataBucketMembers
+  tags:
+    - DemoDataBuckets
+  summary: List of Demo Data Buckets
+  parameters:
+    - description: bucket id
+    - in: path
+      name: bucketID
+      schema:
+        type: string
+  responses:
+    '204':
+      description: A list of demo data buckets
+    '200':
+      description: if sampledata route is not available gateway responds with 200
+    default:
```

Comment:
This was weird but it doesn't look like the UI cares about data being returned besides a status code

Reply:
that is interesting.. it returns `no content` when there's a list of buckets, and an `ok` when the route does **not** exist (and no content is returned otherwise)?

Reply:
This is just an assumption, but I think that the feature intent here is that we add a member to a bucket whenever they try and access the demo data bucket, and so we just make a request to add or remove the user from the bucket without actually caring what the outcome is since we don't actually care to return a list of members. I _think_ the purpose is to enable the demodata bucket for a user? 🤷🏽